### PR TITLE
Cleanups

### PR DESCRIPTION
**Pull request by mmiladi**

### CODE REVIEW

**Comment by mmiladi** [comment]
Do u think whether I can remove or lighten some of these tests to bring everything into the travis 50min lim?
@bgruening 
https://github.com/BackofenLab/GraphClust-2/blob/cleanups/.travis.yml#L26

**Comment by bgruening** [comment]
mh, it passed already above? Is this needed? We can ask the travis support to extend the build time.

**Comment by mmiladi** [comment]
We are back just to the limit, so should be fine.

**Comment by mmiladi** [comment]
Would the last commit be ok :D?

**Comment by mmiladi** [comment]
@bgruening would u please have look

File: Dockerfile [hunk 1]
```diff
@@ -23,8 +23,10 @@ RUN install-tools $GALAXY_ROOT/tools_2.yaml && \
     # rm /export/galaxy-central/ -rf 
 
 RUN install-tools $GALAXY_ROOT/tools_3.yaml && \
-    /tool_deps/_conda/bin/conda clean --tarballs --yes && \
-    rm /export/galaxy-central/ -rf 
+    /tool_deps/_conda/bin/conda clean --tarballs --yes 
+    # && \
+
+RUN rm /export/galaxy-central/ -rf &
```
Comment (bgruening) [inline Dockerfile]: Why two RUN sections?

Comment (mmiladi) [reply Dockerfile]: Was updated before ur comment, strange..

File: Dockerfile [hunk 2]
```diff
@@ -12,16 +12,18 @@ COPY ./assets/tools/graphclust_tools2.yml $GALAXY_ROOT/tools_2.yaml
 COPY ./assets/tools/graphclust_utils.yml $GALAXY_ROOT/tools_3.yaml
 
 RUN install-tools $GALAXY_ROOT/tools.yaml && \
-    /tool_deps/_conda/bin/conda clean --tarballs --yes && \
-    rm /export/galaxy-central/ -rf
+    /tool_deps/_conda/bin/conda clean --tarballs --yes 
+    # && \
+    # rm /export/galaxy-central/ -rf
```
Comment (bgruening) [inline Dockerfile:17]: why have you removed this?

Comment (mmiladi) [reply Dockerfile:17]: It helped reduce build-time for CI. I kept the last rm, to clean-up once at after all 3.

Comment (bgruening) [reply Dockerfile:17]: but this now makes your container bigger ... just saying.

**Comment by mmiladi** [comment]
Hmm. The third rm is not commented and all the three remove same path. So shouldn’t the size be the same? Or not due to creating one layer per RUN..?

**Comment by mmiladi** [comment]
It would have been good if we could have a light option for bioblend tests.

**Comment by bgruening** [comment]
yes, every run creates a layer that is committed and is part of the container.

**Pull request merged**